mini-components/Buttons/discordButton.tsx

```diff
@@ -0,0 +1,17 @@
+"use client";
+const DiscordButton = () => {
+  const handleDiscordClick = () => {
+    window.open("https://discord.gg/B99yGVK5K8", "_blank");
+  };
+
+  return (
+    <button
+      className="bg-transparent text-blue-600 font-semibold py-3 px-6 rounded-lg border border-blue-600 hover:bg-blue-600 hover:text-white transition-colors"
+      onClick={handleDiscordClick}
+    >
+      Discord
```

Comment:
Can't we have one button component and then name passed down by props?

Reply:
@faizm10

Reply:
hmmm, let me try that doing then